src/components/atoms/VisualisationParameterSelector.tsx

```diff
@@ -19,14 +26,25 @@ const VisualisationParameterSelector: React.FC<VisualisationParameterSelectorPro
         }
     };
 
+    const defaultValue = () => {
+        switch (size) {
+            case DashboardItemSize.LARGE:
+                return 'large';
+            case DashboardItemSize.MEDIUM:
+                return 'medium';
+            case DashboardItemSize.SMALL:
+                return 'small';
+        }
+    };
+
     const handleTextChange = (e: React.ChangeEvent<HTMLTextAreaElement>) => {
         setParagraph(e.currentTarget.value);
     };
 
     return (
         <Pane height="100%" display="flex" flexDirection="column">
             <Pane height="32px" marginBottom="1rem" />
-            <SelectField label="Velg størrelse" onChange={handleSizeChange} defaultValue="large">
+            <SelectField label="Velg størrelse" onChange={handleSizeChange} defaultValue={defaultValue()}>
                 <option value="small">Liten</option>
                 <option value="medium">Medium</option>
                 <option value="large">Stor</option>
```

Comment:
Just a question regarding the options elements. Is it possible to set the option value to the DashboardItemSize enum? This feels a little untidy.

Reply:
Probably. I agree

Reply:
I would like to further this idea by using [string enums](https://www.typescriptlang.org/docs/handbook/enums.html#string-enums) to remove the switch statement entirely.

Reply:
Added a new solution now which removes the switch statement.